lib/utils/mongoclient/src/test/keys/server.key

```diff
@@ -0,0 +1,18 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@hsiliev why are private keys and certs being committed in the clear?

Even if they are no longer valid, I'd suggest we use some place holder. Can we do that?